### DISCUSSION TITLE
Is `menu-title` as tidy as it could be?

### DISCUSSION THREAD

**Discussion by longrunningprocess** [saadeghi/daisyui#760] [Q&A]
from https://daisyui.com/components/menu/#menu-with-title:

```html
<ul class="menu bg-base-100 w-56 p-2 rounded-box">
  <li class="menu-title">
    <span>Category</span>
  </li>
  <li><a>Item 1</a></li>
  <li><a>Item 2</a></li>
  <li class="menu-title">
    <span>Category</span>
  </li>
  <li><a>Item 1</a></li>
  <li><a>Item 2</a></li>
</ul>
```

In order to get the appropriate styling on these, a `span` is required.  I was wondering whether it would be possible with the selectors to not require the `span` changing the example above to a little cleaner HTML:

```html
<ul class="menu bg-base-100 w-56 p-2 rounded-box">
  <li class="menu-title">Category</li>
  <li><a>Item 1</a></li>
  <li><a>Item 2</a></li>

  <li class="menu-title">Category</li>
  <li><a>Item 1</a></li>
  <li><a>Item 2</a></li>
</ul>
```

Thanks!

**Comment by saadeghi** [comment]
It is possible to change it but this would be a breaking change and would break existing apps that are using daisyUI.  
I would make it a priority on the next major daisyUI version (`3.0`) but since this change does not add any value or fix any problem, I don't think it's a good idea to apply this change.

**Reply by longrunningprocess** [reply]
that makes perfect sense, thanks for considering it in 3.0.  It definitely was more intuitive for me to not have to add the `span` so maybe a change like this will reduce a little friction for others.

Thanks for all you are doing to make my life so much easier!
